### ISSUE TITLE
How to adjust coloring of "git:"

### ISSUE THREAD

**Issue by stefanscheidt** [michaeldfallen/git-prompt#62]
Hi,

first: thank you very much for the great tool.

second: I'm quite a newbie regarding shell prompt customization, so ...

now: I'm using OS X terminal app with dark solarized color theme, see https://github.com/tomislav/osx-terminal.app-colors-solarized. Within this theme, the "git(...)" part of git-radar ist colored black with bad contrast to the dark backround. I haven't found a way to customize that. Can it be customized?

This may be related to issue #40 and #57 .

Again thanks for the great work!

Regards
Stefan

**Comment by michaeldfallen** [comment]
Recently released v0.4 should allow you to do this. All you need to do is export in your `~/.zshrc` or `~/.bashrc` a new format with your preferred colours in it. 

For bash that would be:

```
GIT_RADAR_FORMAT=" \x01<your colour code>\x02git:(\x01\033[0m\x02%{remote: }%{branch}%{ :local}\x01<your colour code>\x02)\x01\033[0m\x02%{ :changes}"
```

replacing `<your colour code>` with something from [the Bash colour codes list](https://github.com/michaeldfallen/git-radar#bash-colour-codes) e.g. `\033[0;36m` for Cyan.

Note that you need to wrap the code in the `\x01` and `\x02` parts to tell your terminal emulator that these are non printing characters.

For zsh that would be:

```
GIT_RADAR_FORMAT=" %{<your colour code>%}git:(%{$reset_color%}%{remote: }%{branch}%{ :local}%{<your colour code>%})%{$reset_color%}%{ :changes}"
```

replacing `<your colour code>` with something from [the Zsh colour codes list](https://github.com/michaeldfallen/git-radar#zsh-colour-codes) e.g. `$fg[cyan]` for Cyan.

Note that you need to wrap the code in the `%{` and `%}` parts to tell your terminal emulator that these are non printing characters.

**Comment by stefanscheidt** [comment]
Works great! Thank you!
